ceph/rados/rados_bench.py

```diff
@@ -203,7 +202,7 @@ def sequential_read(client, pool_name, **config):
 
         run_name = config.get("run-name")
 
-        seconds = config.pop("seconds")
+        seconds = str(config.pop("seconds"))
         base_cmd.extend(["-p", pool_name, seconds, "seq"])
 
         base_cmd.append(config_dict_to_string(config))
```

Comment:
Suggestion : In future, you may want to add enhancement such that we verify the amount of objects written into the pool, whenever `max-objects` has been provided. To confirm exact no of objects were written.

Reply:
Noted, actually the verification is complex and will fail if the pool already has objects from a previous rados bench or rados put operation. Additionally during the first rados bench iteration, one additional object is written.
I will try to include more verification in this module with another PR